### ISSUE TITLE
error when using search_releases `Argument 7 must be length 1, not 2`

### ISSUE THREAD

**Issue by wjchulme** [dmi3kno/musicbrainz#8]
Hello. The search_release function has started to return an error if it finds certain releases. Best to show by example:

    # search for John Martyn's Piece by Piece using release barcode so it only returns one release - runs without error
    search_releases("042282678829") 
    ##Returning releases 1 to 1 of 1
    
    # search for John Martyn's The Tumbler using release barcode - gives an error
    search_releases("042284243728") 
    ##Returning releases 1 to 1 of 1
    ##Error: Argument 7 must be length 1, not 2

This has only recently started to happen and since the package hasn't been updated recently, presumably it's something on the musicbrainz side. I did a bit of digging and I think it's related to the packaging information (eg jewel case). Sometimes the packaging slot returns two elements instead of one so it can't be put into a (none list-col) column when reformatting into a data frame, hence the error. 

If I use a search term that will return lots of releases it always fails because there's always at least one release with this problem:

    search_releases("best of") 
    ##Returning releases 1 to 25 of 212522
    ##Error: Argument 7 must be length 1, not 2

Can you replicate this error?

**Comment by dmi3kno** [comment]
I pushed an update. These things are relatively easy to fix by modifying `get_main_parser_lst()` to correspond to what API is returning. In this case we just had to adjust expectation to how `packaging` is returned. I understand that `musicbrainz` introduced consistent ID of packaging types, so it is now ID'ed entry with a name. They anticipate that one release can come out in more than one packaging. Would be cool if you let me know should you come across this corner case.
